packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -259,6 +240,15 @@ export default {
             'updateCustomerDetails'
         ]),
 
+        /**
+         * Loads the necessary data to render a meaningful checkout component.
+         *
+         */
+        async initialise () {
+            this.setAuthToken(this.authToken);
+            await Promise.all([this.loadCheckout(), this.loadAvailableFulfilment()]);
```

Comment:
Do you need to handle any possible rejections here from either of the two array items?